earthpy/spatial.py

```diff
@@ -346,22 +347,34 @@ def plot_bands(arr, title=None, cmap="Greys_r",
     --------
     >>>import earthpy.spatial as es
     ...
```

Comment:
Do we also need `import numpy as np` in this example?

Reply:
added!!